modules/vector-sets/commands.json

```diff
@@ -96,7 +96,8 @@
     "command_flags": [
       "WRITE",
       "DENYOOM"
-    ]
+    ],
+    "acl_categories": ["VECTORSET"]
```

Comment:
Add the other categories, e.g., `write` for `VADD` (for the rest as well)

Reply:
Added in 3929738da910ec0c81826ddf8ddd372699613751.
Minor note though, I noticed this not to be the case for the core commands in `src/commands/` (i.e. the commands only listed the group acl category on not those implicitly added because of `command_flags`)

Reply:
Interesting. I don't see a reason not to add them, that may be missing. CC @YaacovHazan

Reply:
command_flags has added implicit acl, so why repeat them in acl_categories?